### PR TITLE
Removed wormtable references from README

### PR DESCRIPTION
**Pull request by jeromekelleher**
Also minor changes to change wormtable related values elsewhere.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,6 @@ python:
   - "2.7"
 
 before_install:
-  - sudo apt-get update -qq
```
Comment (dcolligan) [inline .travis.yml]: why are we removing these lines?

Comment (dcolligan) [inline .travis.yml]: er, I guess we don't need them for anything other than BDB?

**Comment by dcolligan** [comment]
+1

File: .travis.yml (hunk 1 above)
Comment (jeromekelleher) [inline .travis.yml]: Yep.

**Pull request merged**